common/internal-types/Cargo.toml

```diff
@@ -12,10 +12,16 @@ license = "GPL-3.0-only"
 crate-type = ["rlib"]
 
 [dependencies]
+alloy = { workspace = true, default-features = false, features = [
+  "contract",
+  "k256",
+] }
 aquamarine = { workspace = true }
-bloomfilter = { workspace = true }
+bloomfilter = { workspace = true, features = ["serde"] }
+bincode = { workspace = true }
```

Comment:
Why is `bincode` here, if it is not used in the package? Pls run `cargo machete` and verify that all dependencies are really necessary.

Reply:
it's used in various files in the crate.